workers/unity/ProjectSettings/EditorSettings.asset

```diff
@@ -14,8 +14,9 @@ EditorSettings:
   m_EtcTextureFastCompressor: 1
   m_EtcTextureNormalCompressor: 2
   m_EtcTextureBestCompressor: 4
-  m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd
+  m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd;asmdef
   m_ProjectGenerationRootNamespace: 
   m_UserGeneratedProjectSuffix: 
   m_CollabEditorSettings:
     inProgressEnabled: 1
+  m_EnableTextureStreamingInPlayMode: 1
```

Comment:
?

Reply:
We wanted this line - `m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd;asmdef` 

But this one snuck in as well - we can remove? But it always shows up in diffs

Reply:
I guess if Unity really wants that setting, may as well let it have its way.